### DISCUSSION TITLE
How to get the users registered e-mail?

### DISCUSSION THREAD

**Discussion by Conspyre-tv** [supabase/supabase#4669] [Questions]
Creating an "admin" type app, use case is "send e-mail to registered user".

I can make a materialized view of the auth.users table (just email and id should be all we need) & access it directly, that works. What I haven't figured out is how to get the users uuid to look anything up out of that auth table (My thinking... if we can get that uuid at create time, we can save it in the user's table & correlate back to the auth table)

Is this even the right approach? And how do we get that UUID?

**Comment by christianfl** [comment]
I'm not exactly sure what you want to do... Did you try using the helper function `auth.uid()` ? You can use that in SQL queries to get the uid of the requesting user.
